src/redux/configureStore.js

```diff
@@ -0,0 +1,6 @@
+import { configureStore } from '@reduxjs/toolkit';
+import booksReducer from './books/books';
+import categoriesReducer from './categories/categories';
+
+// eslint-disable-next-line no-unused-vars
+const storeData = configureStore({ booksReducer, categoriesReducer });
```

Comment:
- [OPTIONAL] I think It would be better if you export `storeData` variable because you will import it in the future, and Kindly remove this line `// eslint-disable-next-line no-unused-vars` because it's not good practice to make a comment for disabling linter 💯👌

```
const storeData = configureStore({ booksReducer, categoriesReducer });
export default storeData;
```